### DISCUSSION TITLE
Import from Zero-ui

### DISCUSSION THREAD

**Discussion by seanbts** [sinamics/ztnet#174] [Q&A]
Hi,

Is it possible to import the networks and peers from an existing zero-ui installation. 
ztnet is on a new debian server, have a backup of the _data folder which has the networks/peers and secret files.

Thanks

**Comment by PovilasID** [comment]
If anybody is looking for this: 
You can run https://github.com/dec0dOS/zero-ui and ztnet in paralele. 
First you have to mount them both to zerotier's network's volume
Second in controller settings you will have to assign networks to a person to have them available.

**Comment by cscompton** [comment]
Were you able to get this to work? I only get "Error! Controller unreachable" in ZTnet while zero-ui still works fine. ztnet zerotier controller is commented out, attempting to connect to http://zerotier:9993 and localhost:9993. ztnet volume for zeriotier-one is the same path to zero-ui's zerotier-one path to make sure they use the same authtoken and zerotier-one files.

I've tried to get the ip of zu-controller container and tell ztnet to user 'https://zu-controllerIP:9993'

**Reply by sinamics** [reply]
you probably need to set the `allowManagementFrom` in zerotier to allow management from ztnet.
```bash
sudo nano /var/lib/zerotier-one/local.conf
```

Paste in this:
```bash
{
 "settings": {
        "allowManagementFrom": [ "0.0.0.0/0" ]
  }
}
```

Now in ztnet, add zerotier url:  `http://server_ip:9993`, and also add the authtoken.secret.
I just noticed that setting the token will erase the url, so add the token first, then url.

**Reply by cscompton** [reply]
Thanks sinamics! I was able to get it all synced up by placing the docker-compose from zero-ui and ztnet in the same file, and make sure zero-ui elements were in the same docker network. after updating networks and adding names back to the client ID's I removed Zero-ui from the file and down/up docker compose to complete the 'migration'. Having both running at the same time saved a lot of checking zerotier client ID's and adding names back in. I was able to just run down the list from both dashboards side by side and quickly update ztnet names. 

Thanks again!
